component/core/src/main/java/org/exoplatform/social/core/jpa/storage/RDBMSIdentityStorageImpl.java

```diff
@@ -679,12 +679,18 @@ public void processEnabledIdentity(Identity identity, boolean isEnable) {
                                                  long offset,
                                                  long limit,
                                                  boolean forceLoadOrReloadProfile) throws IdentityStorageException {
-    Identity identity = null;
+    Identity viewerIdentity = null;
     if (profileFilter.getViewerIdentity() != null) {
-      identity = profileFilter.getViewerIdentity();
+      viewerIdentity = profileFilter.getViewerIdentity();
     }
     if (OrganizationIdentityProvider.NAME.equals(providerId)) {
-      return getProfileSearchConnector().search(identity, profileFilter, type, offset, limit);
+      List<Identity> identities = getProfileSearchConnector().search(viewerIdentity, profileFilter, type, offset, limit);
+      if (forceLoadOrReloadProfile) {
+        for (Identity identity : identities) {
+          identity.setProfile(loadProfile(identity.getProfile()));
+        }
+      }
+      return identities;
```

Comment:
Revert theses modifications and add the need profile field in ProfileSearchConnector as we did for syncDate and enrollmentDate